docs/articles/samples/IntroJitStatsDiagnoser.md

```diff
@@ -0,0 +1,31 @@
+---
+uid: BenchmarkDotNet.Samples.IntroJitStatsDiagnoser
+---
+
+## Sample: IntroJitStatsDiagnoser
+
+This diagnoser shows various stats from the JIT compiler: 
```

Comment:
It would be great to mention that these stats include entire benchmark run (warmup and BDN boilerplate code)

```suggestion
This diagnoser shows various stats from the JIT compiler that were collected during entire benchmark run (warmup phase and BenchmarkDotNet-generated boilerplate code are included):
```